opus/application/static_media/js/selectMetadata.js

```diff
@@ -290,14 +316,32 @@ var o_selectMetadata = {
         opus.prefs.cols.push(slug);
 
         let label = $(menuSelector).data("qualifiedlabel");
+        let dispUnit = $(menuSelector).data("dispunit");
+        let defaultUnit = $(menuSelector).data("defaultunit");
+
         let info = `<i class="fas fa-info-circle op-metadata-selector-tooltip" title="${$(menuSelector).find(".tooltipstered").tooltipster("content")}"></i>`;
-        let html = `<li id="cchoose__${slug}" class="ui-sortable-handle"><span class="op-selected-metadata-info">&nbsp;${info}</span>${label}<span class="op-selected-metadata-unselect"><i class="far fa-trash-alt"></span></li>`;
+
+        // If a slug has units available, attach the units dropdown menu
+        let unitDropdown = "";
+        if (dispUnit) {
+            unitDropdown += `(<div class="op-units-dropdown dropdown d-inline">
+            <a class="op-${slug}-units-dropdown-toggle dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">${dispUnit}</a>
+            <div class="dropdown-menu op-scrollable-menu" aria-labelledby="dropdownMenuLink">`;
+            let units = $(menuSelector).data("availunits");
+            for (let unit in units) {
+                unitDropdown += `<a class="dropdown-item" data-defaultunit="${defaultUnit}" data-value="${unit}" data-dispval="${units[unit]}" data-slug="${slug}" href="#">${units[unit]}</a>`;
```

Comment:
Adding data-dispval to the possible list of rethinking,   where data-dispValue may be more consistent <at least w/the other data things that are being used for this feature - ie., if we change those, then we should change this as well>